Kontent.Ai.AspNetCore/Webhooks/Models/WebhookNotification.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+namespace Kontent.Ai.AspNetCore.Webhooks.Models
+{
+    /// <summary>
+    /// Root object of a Kontent.ai triggered webhook.
+    /// See <see href="https://kontent.ai/learn/docs/webhooks/webhooks/net">webhooks reference documentation</see> for details.
+    /// </summary>
+    public class WebhookNotification
```

Comment:
Moreover, I would add snapshot test for this model :D

Reply:
added a deserialization test